pool/pool.gno

```diff
@@ -24,13 +24,13 @@ func Mint(
 	recipient std.Address,
 	tickLower int32,
 	tickUpper int32,
-	_liquidityAmount string, // uint128
+	_liquidityAmount string,
```

Comment:
as our rule, should to use liquidityAmount instead of _liquidityAmount

Reply:
fixed https://github.com/gnoswap-labs/gnoswap/pull/403/commits/399b4846836c1e344c8c6beb9788dc958e8a4f67